django_app/redbox_app/redbox_core/models.py

```diff
@@ -64,10 +66,20 @@ class File(UUIDPrimaryKeyBase, TimeStampedModel):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     original_file_name = models.TextField(max_length=2048, blank=True, null=True)
     core_file_uuid = models.UUIDField(null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
```

Comment:
does this need to be persisted?

Reply:
yes, because it gets updated when the file gets used by RAG

Reply:
ah ofc